lib/liquid/nimble_parser.ex

```diff
@@ -68,6 +69,11 @@ defmodule Liquid.NimbleParser do
   defparsec(:decrement, Decrement.tag())
   defparsec(:increment, Increment.tag())
 
+  defparsec(:cycle_group, Cycle.cycle_group())
```

Comment:
Does the parser really need all the parsecs for group/body/values? Because I think the cycle parsec is what composes these.

Reply:
You're right, `group` and `body` can be called like private functions inside `cycle.ex` (corrected) but `values` is recursive and must be defined with `defparsec`